NetworkPkg/Ip6Dxe/Ip6Mld.c

```diff
@@ -193,7 +193,14 @@ Ip6SendMldReport (
   // Fill in MLD message - Report
   //
   MldHead = (IP6_MLD_HEAD *)NetbufAllocSpace (Packet, sizeof (IP6_MLD_HEAD), FALSE);
-  ASSERT (MldHead != NULL);
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (MldHead == NULL) {

```

Comment:
Looks like `NetbufFree (Options);` should be in this block.

Reply:
These `NetbufAllocSpace()` calls appear to be carving out space from an already allocated buffer so I'll drop my comment on them.